pkg/secrets-store/nodeserver_test.go

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
setting up initial unit test framework so we can expand the coverage in subsequent PRs